Mage/src/main/java/mage/abilities/condition/common/ThresholdCondition.java

```diff
@@ -0,0 +1,33 @@
+package mage.abilities.condition.common;
+
+import mage.abilities.Ability;
+import mage.abilities.condition.Condition;
+import mage.game.Controllable;
+import mage.game.Game;
+import mage.players.Player;
+
+import java.util.HashSet;
+import java.util.Optional;
+
+/**
+ * @author TheElk801
+ */
+public enum ThresholdCondition implements Condition {
+    instance;
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        return Optional
+                .ofNullable(source)
+                .map(Controllable::getControllerId)
+                .map(game::getPlayer)
+                .map(Player::getGraveyard)
+                .map(HashSet::size)
+                .orElse(0) > 0;
+    }
+
+    @Override
+    public String toString() {
+        return "there are seven or more cards in your graveyard";
```

Comment:
for all threshold cards, the current oracle wording is "seven or more cards are in your graveyard"

(this wording is only used by Deep-Sea Terror and Illuminate History, which don't have the threshold ability word)

Reply:
I tried to copy what the previous value would do, I figure it's not worth worrying too much about the text discrepancies until after all the errata is applied anyway

Reply:
yeah no worries, I had just seen a couple dozen discrepancies on cards with threshold for a long time and never been motivated enough to fix them properly, and this is the perfect opportunity